src/hooks/useToast.ts

```diff
@@ -0,0 +1,36 @@
+import { useSetRecoilState } from "recoil";
+import toastState from "@/stores/atoms/toastState";
+
+export default function useToast() {
+  const setToastList = useSetRecoilState(toastState);
+
+  const addSuccessToast = (message: string) => {
+    setToastList((prev) =>
+      prev.concat({
+        id: Date.now(),
+        type: "success",
+        message,
+      }),
```

Comment:
```suggestion
      [
        ...prev,
        {
          id: Date.now(),
          type: "success",
          message,
        }
      ]
```
요즘은 concat보다 구조분해할당을 더 많이 쓰는 것 같습니다!

Reply:
완료